.autoroller-preupload.sh

```diff
@@ -29,3 +29,8 @@ ninja -C "$PROFILE_PATH" licenses_cpp
   --licenses_path="$LICENSES_PATH" \
   --data_dir="$DATA_PATH" \
   --v=1
+
+cd "$REPO_PATH/engine/src"
+./tools/dart/create_updated_flutter_deps.py
+cd "$REPO_PATH"
+gclient sync
```

Comment:
@aam I asked the question in the issue but are you sure we want to do this as part of the preupload script?  Do we want to catch failures here or later downstream?

Reply:
I expected this repo to be trashed after the upload though, so this would be wasted work beyond providing a check that it works, no?

Reply:
I expect that this `.autoroller-preupload.sh` is executed while roll is actually being executed by the bot. So the bot will commit whatever this script updates(DEPS as well as license files, not sure what else). So effectively flutter `DEPS` file is updated in two steps during a roll: first when roller picks up dart revision, second - when `.autoroller-preupload.sh` updates it based on `DEPS` from `third_party/dart`